### ISSUE TITLE
[regression/8.0.0] [Android] Border inside frame cause dont render content

### ISSUE THREAD

**Issue by LaBoss** [dotnet/maui#18526]
### Description

Hello, when i put border inside a Frame -> Grid -> Border the content is empty dont render

**With border**
![Screenshot_1699193855](https://github.com/dotnet/maui/assets/5363187/dd57aa50-3fbc-4ad7-a47f-f5fabeee254d)

Without border
![Screenshot_1699193823](https://github.com/dotnet/maui/assets/5363187/699a109a-77ff-4a93-b602-a6055d69454f)

XML Code
```
 <VerticalStackLayout
  Spacing="25"
  Padding="30,0"
  VerticalOptions="Center">
     <Frame>
         <Grid RowDefinitions="auto, auto, auto, auto">
             <Border Stroke="#C49B33"
                     StrokeThickness="4"
                     StrokeShape="RoundRectangle 40,0,0,40"
                     Background="#2B0B98"
                     Padding="16,8"
                     HorizontalOptions="Center" Grid.Row="0">
                 <Label Text=".NET MAUI"
                        TextColor="White"
                        FontSize="18"
                        FontAttributes="Bold" />
             </Border>

             <Label Grid.Row="1"
                      Text="Login"
                      FontSize="18"
                      HorizontalOptions="Center" />

             <Button Grid.Row="2"
                      Text="Login"
                      Clicked="OnLogin"
                      HorizontalOptions="Center" />

             <Button Grid.Row="3"
                     Text="Settings"
                  Clicked="OnSettings"
                  HorizontalOptions="Center" />
         </Grid>
     </Frame>
 </VerticalStackLayout>
```

### Steps to Reproduce

1. Create MAUI .NET 8 Project
2. On page Add VerticalStackLayout, Frame, Grid, Border
3. The content is not displayed

### Link to public reproduction project repository

https://github.com/LaBoss/MAUITestes.git

### Version with bug

8.0.0-rc.2.9511

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

7.0.96

### Affected platforms

Android, I was *not* able test on other platforms

### Affected platform versions

Android 11

### Did you find any workaround?

Dont use border

**Comment by XamlTest** [comment]
Verified this on Visual Studio Enterprise 17.8.0 Preview 7.0(8.0.0-rc.2.9530). Repro on Android 14.0-API34, not repro on Windows 11 with below Project:
[PrismBug.zip](https://github.com/dotnet/maui/files/13294333/PrismBug.zip)

**Comment by GerhardMauerberger** [comment]
I thought this problem occurs only with Popup. (https://github.com/CommunityToolkit/Maui/issues/1585)
I checked it once again and it is more serious than I thought. We cannot use any Frame for Popup and for ContentPage.

**Comment by smardine** [comment]
Hi, it's not only Popup relevent, it happen as soon as you use a frame aroud a AvatarView from CommunityToolkit.Maui too
https://github.com/CommunityToolkit/Maui/issues/1600

**Comment by mikeysouthwell** [comment]
+1 for this. I can't get content inside a border to render when it is inside a frame.

With the example below, the label or anything else you place inside the internal border would not be visible on the screen.

This has been tested on the latest version of Android and Visual Studio.

Installed Workload Id      Manifest Version       Installation Source
---------------------------------------------------------------------
maui-windows               8.0.3/8.0.100          VS 17.8.34525.116
maccatalyst                17.2.8004/8.0.100      VS 17.8.34525.116
android                    34.0.43/8.0.100        VS 17.8.34525.116
ios                        17.2.8004/8.0.100      VS 17.8.34525.116

```MAUI
<border>
    <frame>
        <border>
           <label />
        </border>
    </frame>
</border>
```

**Comment by PureWeen** [comment]
Duplicate of #18755

**Comment by PureWeen** [comment]
This fix should be available on the nightly feed as version `8.0.61-ci.net8.24319.3`. 

Can you test with the latest nightly build? 
https://github.com/dotnet/maui/wiki/Nightly-Builds
